src/transformers/modeling_utils.py

```diff
@@ -3432,6 +3438,7 @@ def from_pretrained(
                     os.path.join(pretrained_model_name_or_path, subfolder, _add_variant(WEIGHTS_NAME, variant))
                 ):
                     # Load from a PyTorch checkpoint
+                    print("pytorch checkpoint")
```

Comment:
If we're emitting to the terminal, it should be using `logging`. I don't think this would be very informative for the user however

```suggestion
```